system_strings/sysmes_text.en

```diff
@@ -657,21 +657,21 @@ Encounter
 ...Stay cautious but alert.
 Passing out in the street
 ...Confront her as an enemy.
-八日目・朝。
-今後のご予定は。
-遠野志貴の泣かせ方
-浅はかな夢
-シエルとノエル
-先輩は保健室を去った。
-シエルの不審な行動
-気になる。先輩を追いかけよう。
-戻りましょう、先輩。
-シエルさん、インドですよ。
-嘘は言ってない。
-アルクェイドの方がほっとけない。
-あの人は、大切な先輩だ。
-主に祈りは届きませり
-1989年
+8th day - Morning
+What's the plan from here?
+The bane of my existence
+Shallow, fleeting dream
+Ciel and Noel
+Senpai leaves the infirmary.
+Senpai's suspicious actions.
+I'm curious, I'll follow Senpai.
+Let's head back, Senpai.
+Ciel-san in India.
```

Comment:
This is the scene in the curry shop. Might just need to be something else entirely.